packages/vendure-plugin-stripe-subscription/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@pinelab/vendure-plugin-stripe-subscription",
-  "version": "2.8.0",
+  "name": "vendure-plugin-stripe-subscription",
+  "version": "2.8.1",
   "description": "Vendure plugin for selling subscriptions via Stripe",
   "author": "Martijn van de Brug <martijn@pinelab.studio>",
   "homepage": "https://pinelab-plugins.com/",
-  "repository": "https://github.com/Pinelab-studio/pinelab-vendure-plugins",
+  "repository": "https://github.com/givelegacy/pinelab-vendure-plugins",
```

Comment:
@mshoaib10 Could you change this back to `"repository": "https://github.com/Pinelab-studio/pinelab-vendure-plugins"`? Since that is the original repository.